rfc/rfc007-api-stability.md

```diff
@@ -0,0 +1,93 @@
+---
+RFC: 007
+Author: Andrew Gerrand <adg@golang.org>
+Status: Draft
+---
+
+# API Stability Policy
+
+## Motivation
+
+The gokit project depends on code maintained by others.
+This includes the Go standard library and sub-repositories
+and other external libraries.
+
+The Go project provides stability guarantees, but the other external libraries
```

Comment:
"The Go project" should probably be replaced with something more specific, like "The Go language and standard library", since the subrepos (x/tools and friends) don't have any stability guarantees (yet).